Multisig/UI/App/ViewState.swift

```diff
@@ -20,6 +20,9 @@ class ViewState: ObservableObject {
         state = to
     }
 
+    @Published
+    var presentedSafeTxHash: String?
```

Comment:
All good, but I think we need to wrap it in some structure

Reply:
we'll do it in another pass over the whole notifications code next week